doc/source/release-notes/Percona-Server-8.0.18-9.rst

```diff
@@ -0,0 +1,78 @@
+.. rn:: 8.0.18-9:
+
+================================================================================
+|Percona Server| |release|
+================================================================================
+
+|Percona| announces the release of |Percona Server| |release| on |date| (downloads are available `here <https://www.percona.com/downloads/Percona-Server-8.0/>`__ and from the `Percona Software Repositories <https://www.percona.com/doc/percona-server/8.0/installation.html#installing-from-binaries>`__).
+
+This release includes fixes to bugs found in previous releases of |Percona
+Server| 8.0.
+
+|Percona Server| |release| is now the current GA release in the 8.0 series. All
+of |Percona|’s software is open-source and free.
+
+Percona Server for MySQL 8.0 includes all the `features available in MySQL
+8.0.18 Community Edition
+<https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-18.html>`__ in
+addition to enterprise-grade features developed by Percona.
+
+Bugs Fixed
+================================================================================
+
+- Setting the ``none`` value for :variable:`slow_query_log_use_global_control`
+  generates an error. Bugs fixed :psbug:`5813`.
+
+- If `pam_krb5
+  <https://docs.oracle.com/cd/E88353_01/html/E37853/pam-krb5-7.html>`__ allows the
+  user to change their password, and the password expired, a new password may
+  cause a server crash. Bug fixed :psbug:`6023`.
+
+- A check if any temporary tables should be logged to binlog may cause the
```

Comment:
This happened only in debug build (incorrect assertion was triggered)

Reply:
updated.